### ISSUE TITLE
V3-repo-wide Report of completed exercises

### ISSUE THREAD

**Issue by mikedamay** [exercism/v3-dashboard#45]
It would be useful to have a report available of exercise names together with the language for all completed or substantially completed exercises in the V3 repo in concept/exercise name order. 

 In the event that we start to push parts of exercises to the repo rather than complete exercises then we should favour including anything with an _instrutions.md_ even if it is incomplete.

Nice to have would be to have links to the _instructions.md_ file.  If the story name is available I suppose it would add some information but I don't think it is essential.

The purpose of the report is to allow a v3 track contributor to quickly inspect existing exercises to gain inspiration before starting work on a new exercise.

I don't know what the norm is for these sort of pages but if there are any constraints then a daily update would probably be sufficient.

The report can be used in conjunction with the curated [stories](https://github.com/exercism/v3/tree/master/reference/stories) document.
